examples/replist_changes_getting.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import time
+import uuid
+from os import environ
+from typing import cast
+
+from cybsi.api import APIKeyAuth, Config, CybsiClient
+from cybsi.api.observable import EntityView
+from cybsi.api.pagination import Cursor, Page
+from cybsi.api.replist import EntitySetChangeView
+
+if __name__ == "__main__":
+    api_key = environ["CYBSI_API_KEY"]
+    api_url = environ["CYBSI_API_URL"]
+
+    auth = APIKeyAuth(api_url=api_url, api_key=api_key)
+    config = Config(api_url, auth, ssl_verify=False)
+
+    with CybsiClient(config) as client:
+        wait_on_empty_changes_sec = 10
+        replist_uuid = uuid.uuid4()
+
+        cursor_for_changes: Cursor
+        entities_page: Page[EntityView]
+        # By requesting entities list we get the last cursor
+        # to get reputation list changes.
+        # The change cursor starts with the latest change of entity in list.
+        # It is recommended that you first obtain all pages
```

Comment:
Может ужесточим? Пусть будет "обязательно". И привлечем больше внимания к этому комментарию? 
```suggestion
        # WARNING: It is mandatory that you first obtain all pages
```

Reply:
Ага, сделано)